tasks.py

```diff
@@ -0,0 +1,85 @@
+import invoke
+from invoke import task
+import os
+from yaksh.settings import SERVER_POOL_PORT
+
+SCRIPT_DIR = os.path.abspath(os.path.dirname(__file__))
+TARGET_CONTAINER_NAME = 'yaksh_code_server'
+SRC_IMAGE_NAME = 'fossee/yaksh_codeserver'
+
+def create_dir(path):
+    if not os.path.exists(path):
+        os.makedirs(path)
+
+@task
+def setupdb(ctx):
+    print("** Setting up & migrating database **")
+    ctx.run("python manage.py migrate")
```

Comment:
Moderators cannot be created unless the user creates admin and invoke doesnt have a create admin command.